### PR TITLE
Fixed names in the haven_onDemand content

### LINKED ISSUE #601: Align the haven_onDemand content to our best practices
The haven_onDemand content is not aligned (best practices - naming conventions - snake_case) with the rest of the content. Please refactor it to be consistent.

### PR DESCRIPTION
**Pull request by dashapaster**
fixes #601 
change names in content

### CODE REVIEW

**Comment by dashapaster** [comment]
@mihaitusa please review.

**Comment by Bonczidai** [comment]
hi @dashapaster -- from builder output:

```console
07:48:04 [INFO] 1 test cases have missing test flow references:
07:48:04 [INFO] For test case: testsendTranscriptToMail testFlowPath reference not found: io.cloudslang.haven_onDemand.voice_recognision.sendTranscriptToMail
```

it means that this test case is supposed to run a flow which does not exist, please take a look

**Comment by dashapaster** [comment]
Hi @Bonziday, it's strange,  because i've fixed it in last commit. I check it

**Comment by dashapaster** [comment]
@Bonczidai , where i can see  this?

in circleci i found just this error:
05:11:43 [ERROR] Following 1 tests failed:
05:11:43 [ERROR] - Test case failed: testJenkins_clone_job_for_branch_Success - Tests that clone job for branch meets its intended purpose

and  about my test:
05:10:57 [INFO] Skipping test: testsendTranscriptToMail because it is not in active test suites

**Comment by Bonczidai** [comment]
I took it from `master`, you are right -- it's already fixed on this PR :ok_hand:

File: content/io/cloudslang/haven_onDemand/speech_recognision.prop.sl [hunk 1]
```diff
@@ -0,0 +1,11 @@
+namespace: io.cloudslang.haven_onDemand
```
Comment (mihaitusa) [inline content/io/cloudslang/haven_onDemand/speech_recognision.prop.sl]: @dashapaster  I mean here for example to rename haven_onDemand folder to haven_on_demand....

Comment (dashapaster) [inline content/io/cloudslang/haven_onDemand/speech_recognision.prop.sl]: @mihaitusa, yes i udrerstend it, i fixed all names, but when i tried to fix name of folder i get error, like dont have permission, i will try again

**Comment by mihaitusa** [comment]
@dashapaster :+1:  everything looks OK now.

**Pull request merged**